src/App.jsx

```diff
@@ -0,0 +1,48 @@
+import './App.css';
+import React, { useState } from 'react';
+import axios from 'axios';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Cookies from 'js-cookie';
+import Login from './pages/Login.jsx';
+import Landing from './pages/Landing.jsx';
+import Signup from './pages/Signup.jsx';
+import { BackendUrlProvider } from './components/BackendUrl.jsx';
+import PrivateRoutes from './components/PrivateRoutes.jsx';
+
+// make sure that axios always sends the cookies to the backend server
+axios.defaults.withCredentials = true;
+
+const BACKEND_URL = process.env.REACT_APP_BACKEND_URL || 'http://localhost:3004';
```

Comment:
this doesn't work right? because the env variable is wrong so it defaults to the second one
```suggestion
const BACKEND_URL = process.env.BACKEND_URL || 'http://localhost:3004';
```

Reply:
Ya, think so. Renamed to REACT_APP_BACKEND_URL in .env